Bottom: 01a9ec2965de01b9ac64904e14e86302135af99c
Top:    8bc3ce72ca2e5b0dbe083fdb3e39f50ea05a3bf4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-31 01:22:32 +0100

Refresh of give-the-hids-control-over-att

---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index 76400e4..ba1c4ad 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -780,6 +780,10 @@ ghid_invalidate_all ()
 void
 ghid_notify_crosshair_change (bool changes_complete)
 {
+#warning FIXME
+  if (gport->drawing_area == NULL)
+    return;
+
   if (changes_complete)
     CrosshairOn ();
   else
